src/routes/home/Home.js

```diff
@@ -1,46 +1,30 @@
-/**
- * React Starter Kit (https://www.reactstarterkit.com/)
- *
- * Copyright © 2014-2016 Kriasoft, LLC. All rights reserved.
- *
- * This source code is licensed under the MIT license found in the
- * LICENSE.txt file in the root directory of this source tree.
- */
-
+import Feed from '../../components/Feed'
 import React, {PropTypes} from 'react'
 import withStyles from 'isomorphic-style-loader/lib/withStyles'
 import s from './Home.scss'
-import Story from '../../components/Story'
-import StoryInput from '../../components/StoryInput'
+import {connect} from 'react-redux'
+import {loadStories} from '../../actions/stories'
 
 const title = 'Rate Issues'
 
-function Home ({stories}, context) {
+function Home ({stories,loadStories}, context) {
   context.setTitle(title)
+  loadStories({stories: stories})
```

Comment:
This should be changed in the future. To load the stories we should place somewhere associated with the component mount hook. This is a case where we will be dealing with cache like "has the stories changed ? if not, give me the cached stories"